spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1295,6 +1421,8 @@ public interface ProcessObserverMethod<T, X> {
 * `getObserverMethod()` returns the `ObserverMethod` object that will be used by the container to call the observer method.
 * `addDefinitionError()` registers a definition error with the container, causing the container to abort deployment after bean discovery is complete.
 * `setObserverMethod()` replaces the `ObserverMethod`.
+`configureObserverMethod()` returns an `ObserverMethodConfigurator` (as defined in <<observer_method_configurator>>) initialized with the `ObserverMethod` processed by the event to easily configure the  `ObserverMethod` which will be used to replace the original one at the end of observer invocation.
+The method always return the same `InjectionPointConfigurator`.
```

Comment:
And fix formatting please. I cannot see list bullet for `configureObserverMethod()` in my html spec.